app/frontend/src/store/modules/apiAccess.js

```diff
@@ -30,7 +30,16 @@ export default {
     ephemeralPasswordRSAKey: state => state.ephemeralPasswordRSAKey,
     generatedClient: state => state.generatedClient,
     generatedPassword: state => state.generatedPassword,
-    step: state => state.step
+    step: state => state.step,
+    tokenEndpoint(state) {
+      let prefix = 'sso';
+      if (state.environment.toLowerCase() === 'dev') {
+        prefix += '-dev';
+      } else if (state.environment.toLowerCase() === 'test') {
+        prefix += '-test';
+      }
```

Comment:
Could do something like this to remove the need to do multiple conditionals - just leverage state.environment directly?
```js
if (['dev','test'].includes(state.environment.toLowerCase())) {
  prefix += `-${state.environment.toLowerCase()}`
}
```